demo/views/pages/component/component-api/component-api.js

```diff
@@ -68,10 +70,14 @@ class ComponentAPI extends React.Component {
   _default = (prop) => {
     let value = this.props.definition.getIn(['defaultProps', prop]);
 
-    if (typeof value === "boolean") {
+    if (typeof value === 'boolean') {
       value = String(value);
     }
 
+    if (typeof value === 'function') {
+      value = value.toString();
```

Comment:
Calling `toString()` on a `function` will [return the source code of the function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/toString#Return_value) (which could potentially be a very long string), which will be displayed in the `<TableCell>` by `_buildRows()` above. Is that definitely the functionality we want here?

Reply:
Makes sense, as discussed I've updated this to just print "function"